oracle/packages/node/.chainConfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Seems like magic to me, should this be documented somewhere (if it isn't)?

Reply:
I'll add it to the readme in the explanation of the `fuel-core ...` command.  Its purpose is to configure the local fuel node with a wallet `0x09c...` and gives the wallet a lot of the base asset.  I'll add documentation about the  specified wallet the secret_key in `.env.example`